src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmExportService.java

```diff
@@ -18,13 +18,15 @@
 import static java.nio.charset.StandardCharsets.ISO_8859_1;
 import static org.slf4j.LoggerFactory.getLogger;
 
-import java.io.IOException;
+import java.io.*;
```

Comment:
I recently added a intellij code style configuration to the box-c project which you will want to import for the migration utility as well. You can find it here:
https://github.com/UNC-Libraries/box-c/tree/main/etc/ide_setup
Its often considered a best practice to avoid star imports, since it pulls a lot of extra stuff into your code that you likely aren't using. I know Intellij generated this line, it should hopefully not do that after setting up the code style thing